packages/space-opera/src/redux/space_opera_base.ts

```diff
@@ -38,6 +38,8 @@ export interface State {
   animationNames: string[];
   gltfJsonString: string;
   edits: GltfEdits;
+  // A copy of the original, so we can revert individual properties.
+  origEdits: GltfEdits;
```

Comment:
Not sure these are edits, strictly speaking, but I see that's what we call the type.